device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/th2-a7260cx3-64-64x100G.config.bcm

```diff
@@ -1011,3 +1011,5 @@ serdes_preemphasis_114=0x174507
 serdes_preemphasis_115=0x184606
 serdes_preemphasis_116=0x103706
 serdes_preemphasis_117=0x133c06
+
+mmu_init_config="MSFT-TH2-Tier1"
```

Comment:
C64 profile can also be Tier0, how to handle that?

Reply:
Good catch. Added T0 config.